src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -131,6 +154,8 @@ const ActionsPane = (props: {contentHeight: string}) => {
   const applySelectedResource = useCallback(() => {
```

Comment:
Might need to find a better name for this function and add `selectedPath` to the useCallback deps